[2024-02-02 04:24:16,632] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-02T04:24:11.255303+00:00 [queued]>
[2024-02-02 04:24:16,637] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-02T04:24:11.255303+00:00 [queued]>
[2024-02-02 04:24:16,637] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 04:24:16,638] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-02 04:24:16,638] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 04:24:16,645] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): import_file_to_mysql> on 2024-02-02 04:24:11.255303+00:00
[2024-02-02 04:24:16,648] {standard_task_runner.py:52} INFO - Started process 1676 to run task
[2024-02-02 04:24:16,650] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'import_file_to_mysql', 'manual__2024-02-02T04:24:11.255303+00:00', '--job-id', '338', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpd7makclx', '--error-file', '/tmp/tmp5n1810mt']
[2024-02-02 04:24:16,651] {standard_task_runner.py:80} INFO - Job 338: Subtask import_file_to_mysql
[2024-02-02 04:24:16,686] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-02T04:24:11.255303+00:00 [running]> on host c3bb523480bf
[2024-02-02 04:24:16,729] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=import_file_to_mysql
AIRFLOW_CTX_EXECUTION_DATE=2024-02-02T04:24:11.255303+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-02T04:24:11.255303+00:00
[2024-02-02 04:24:18,979] {logging_mixin.py:115} INFO - Successfully install pandas
[2024-02-02 04:24:21,562] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-02 04:24:24,298] {logging_mixin.py:115} INFO - Successfully install pymysql
[2024-02-02 04:24:24,408] {python.py:173} INFO - Done. Returned value was: import to mysql
[2024-02-02 04:24:24,426] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=import_file_to_mysql, execution_date=20240202T042411, start_date=20240202T042416, end_date=20240202T042424
[2024-02-02 04:24:24,452] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-02 04:24:24,475] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
